andela/wger-kronos
# #166185734 Relation between workout log and session

### What does this PR do?
- Adding a real foreign key from the logs to the session
### Description of Task to be completed?
Right now the workout logs and the workout session are only linked through the date (for sessions this value is unique and enforced by the DB). This presents in some places a problem since it's not possible to just follow a foreign key relationship with the ORM like in other places.

To fix this either:

a) see if it's possible to define the date as the foreign key in Django
or
b) check if adding a real foreign key from the logs to the session makes sense
### How should this be manually tested?
- Switch to this branch and start the server.
- Run the migrations using `python manage.py migrate`
- Go to this URL(`http://127.0.0.1:8000/en/workout/overview`) and try creating the workout to check whether the functionality still works after changing this relationship.
### What are the relevant pivotal tracker stories?
[#166185734](https://www.pivotaltracker.com/story/show/166185734)
